Corvus.Extensions.Samples/IEnumerableExtensionsSample.dib

```diff
@@ -107,7 +109,9 @@ return list.HasMinimumCount(4);
 
 #!markdown
 
-This is an efficient implementation of `enum.Any() && enum.All(predicate)` that avoids starting the enumeration twice. It determines if the collection is non-empty, and that every element also matches some predicate.
+This is an efficient implementation of the built-in LINQ operators [Any()](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.any?view=net-6.0) && [`All(predicate)`](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.all?view=net-6.0) that avoids starting the enumeration twice. It determines if the collection is non-empty, and that every element also matches some predicate.
```

Comment:
I think we should emphasize that it's an efficient implemenation of **the combination** of the `Any` and `All` operators.

If either `Any` or `All` on their own do what you need, you can use them—they are perfectly efficient. It's only if you needed the combination (because you don't like the "vacuous truth" feature of `All`) that this is more efficient.

Reply:
Done.